src/components/Login/Login.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from "react";
+import { useDispatch } from "react-redux";
+import { fetchSessionAndGetUser, fetchToken } from "./LoginApi/services/services";
+import { userRequestAction } from "./userReducer/actionCreators/actionCreators";
+import { LoginForm } from "./LoginForm/LoginForm";
+import { LoginSupport } from "./LoginSupport/LoginSupport";
+import { Avatar, Typography, Box, Container } from "@mui/material";
+import LockOutlinedIcon from "@mui/icons-material/LockOutlined";
+import { UserActions } from "./types/user";
+import { Dispatch } from "redux";
+
+const Login: React.FunctionComponent = () => {
```

Comment:
1. move export here
2. React.FC would be enough

Reply:
done